modules/react/icon/stories/Icons.stories.tsx

```diff
@@ -117,7 +126,7 @@ export const GraphicStory = {
   component: Graphic,
   render: () => (
     <div className="story">
-      <Graphic src={graphicExample} />
+      <Graphic src={graphicExample} cs={{width: 40}} />
```

Comment:
should it be `px2rem(40)`

Reply:
i was just testing that the CS prop works.  Will update!